pytom/agnostic/score.py

```diff
@@ -23,7 +23,7 @@ def Vol_G_Val(volume, value):
         p = peak(value)
         value = value[p]
 
-    if volume.__class__ == pytom_volume.vol:
+    if volume.__class__ == pytom.lib.pytom_volume.vol:
         p = peak(volume)
         volume = volume[p]
 
```

Comment:
```suggestion
```

pytom_volume is not imported here so this statement would result in 'pytom not defined' error. It should only do the above if statement that checks for an ndarray.

I guess all the docstrings below are also wrong because they mention pytom_volume. I guess they were just copied from pytom_volume code. Too much effort to update it now though